components/brave_rewards/resources/ui/components/walletWrapper/index.tsx

```diff
@@ -238,7 +238,6 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
       return
     }
 
-    // tslint:disable
     const isWindows = window.navigator.platform === 'Win32'
```

Comment:
Hmm do we think this should complain in eslint like it did with tslint? That API is marked as deprecated.

Reply:
Since this is always going to be upstreamed from chromium then perhaps we can explicitly mark this as deprecated in eslint. 

As more browsers haven't yet have the alternate api ready ideally we shouldn't limit the devs.

Reply:
Let's create / add to an exiting issue to re-enable the deprecation warnings.

Reply:
https://github.com/brave/brave-browser/issues/19382